### ISSUE TITLE
Custom dataset annotation and feed to model

### ISSUE THREAD

**Issue by shreyasroshan** [deepdoctection/deepdoctection#197]
1) How to annotate the data? Right now am using Label studio for annotation. In which I have a single category "table"
2) What should be the format (json, csv, xml) of the annotated file that has to be fed?
3) Where should we feed the annotated data and what the other files or pieces that has to changed for custom dataset which I intend to do?
4) Also there aren't any tutorials too to check the various behaviours of deepdoctection

**Comment by JaMe76** [comment]
1. **deep**doctection does not provide an annotation tools. Therefore, any tool can be used. Ground truth annotation, however must be transformed.
2. There is no format restriction as long as the ground truth data is converted into **deep**doctection intrinsic format. Publaynet dataset has ground truth in COCO format, i.e. of JSON type. PubTables1M is in Pascal-VOC format, i.e. of XML-type.
3. There is a class `CustomDataset` available that helps feeding the data to the model for training. There are also various training scripts available. Docs can be found [here](https://deepdoctection.readthedocs.io/en/latest/tutorials/datasets/).
4. Not quite sure what is meant with various behaviours. But tutorials for many topics can be found [here](https://deepdoctection.readthedocs.io/en/latest/tutorials/get_started_notebook/) .
